### PR TITLE
Documentation update for Linux users building from source.

### PR DESCRIPTION
**Pull request by domagalski**
I put a note in the readme for any Linux user that wants to build from source on how to do so.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ Once Chrome and Java are installed, use your file manager to navigate to the sco
 
 * Apple users: Run scoreboard-Mac.app to start the script. If you are unable to start it, you may have to download and run the [startscoreboard-mac.zip package](https://drive.google.com/file/d/0B2fZmT3bqA9oS25KNHphZjFFcWs/view?fref=gc&dti=480408282040949) to get the scoreboard started.
 
-* Linux users: Run scoreboard-Linux.sh to start the script. If you are unable to start it, you may have to allow script files to be executable as programs.
+* Linux users: Run scoreboard-Linux.sh to start the script. If you are unable to start it, you may have to allow script files to be executable as programs. The ``lib/crg-scoreboard.jar`` file can be built from source by installing ``apache-ant`` and running ``ant compile`` from this directory.
```
Comment (brian-brazil) [inline README.md]: On Ubuntu the package is `ant`, it's probably best to just link to http://ant.apache.org/

**Comment by domagalski** [comment]
@brian-brazil you're right, and I've now done that.

**Comment by brian-brazil** [comment]
:+1:

**Comment by rogerharkavy** [comment]
@domagalski I also added this information to the wiki, which is where we have been copying material from for the README file. Thanks!

**Comment by brian-brazil** [comment]
Can you resolve the conflicts please?

**Comment by rogerharkavy** [comment]
@domagalski Since this issue was opened, we created a new topic in the wiki about building from source. I put your note at the bottom because I'm not sure where it belongs. @brian-brazil and @jaredquinn since you have both done builds recently, please also review this information and revise as necessary:

https://github.com/rollerderby/scoreboard/wiki/Building-the-Scoreboard-from-Source

I'm not sure what happened with my previous edits so I edited README again, renamed it to README.MD and copied the content from the wiki into the file. Created a new pull request.

**Comment by brian-brazil** [comment]
Okay, so we can close this PR and just do the rename then?

**Comment by rogerharkavy** [comment]
@brian-brazil It's not just the rename, the content of the file is new.

**Comment by brian-brazil** [comment]
The wiki page you linked appears to already cover this content. How do you propose we handle this?

**Comment by rogerharkavy** [comment]
However you want to handle it is fine with me, as long as the README gets renamed to README.md and the content that I added to it today gets into the final build.

**Comment by frank-weinberg** [comment]
Closing this, as it is superseded by #76.

**Pull request closed without merging**